versioned_docs/version-1.1.0/index.md

```diff
@@ -34,7 +34,7 @@ Serverpod Insights is a companion app bundled with Serverpod. It allows you to a
 
 :::info
 
-Download the latest version here: __[Serverpod Insights](insights)__. It is compatible with Serverpod version 1.0.x. Always use the same version of Serverpod Insights as for the framework itself.
+Download the latest version here: __[Serverpod Insights](tools/insights)__. It is compatible with Serverpod version 1.0.x. Always use the same version of Serverpod Insights as for the framework itself.
```

Comment:
Question: When testing this the reference now jumps to the newest version of the documentation if the link is used. Meaning that the visitor will leave the 1.x.x version of the docs and jump to 2.0.0 (currently). Was that intentional?

Reply:
Same for 1.1.1 and 1.2.0.

Reply:
Can't seem to fix this. But may not be a big issue.